deployment/helm/synopsys-alert/README.md

```diff
@@ -29,36 +43,63 @@ $ helm install . --name <name> --namespace <namespace>
 
 ## Installing the Chart -- Helm 3
 
-#### Create the Namespace and TLS Secrets
+### Create the Namespace and TLS Secrets
 ```console
 $ kubectl create ns <namespace>
 ```
-#### Configure your Black Duck Instance
+### Configure your Alert Instance
 Modify the values.yaml file or pass in values to `helm install` with --set.  
+Please see the [Configuration](#configuration) section for more details.
 
-#### Install the Black Duck Chart
-```
+### Install the Alert Chart
+```console
 $ helm install <name> . --namespace <namespace>
 ```
 
-## Quick Start with Helm 3
+### Quick Start with Helm 3
 #### Step 1
 Navigate to the alert-helm chart repository in your terminal
-```
+```console
 $ cd <path>/alert-helm
 ```
 
 #### Step 2
-```
+```console
 $ kubectl create ns myalert
 ```
 
 #### Step 3
 Deploy Alert
-```
+```console
 $ helm install myalert . --namespace myalert
 ```
 
+## Finding Alert External Port
+ Once Alert has been deployed if the `exposeui` parameter is true, then Alert will be available via an exposed port.
+ To determine the port to access the Alert UI execute the following command:
+ ```console 
+ $ kubectl -n <NAMESPACE> get services
+ ```
+ From the output find the Alert exposed service. This service will be your Alert installation name with the `-exposed` suffix in the name. 
+ If the installation name is 'myalert' then there will be a service `myalert-exposed` in the list of services.
+ 
+ It will display a port with the following format:
+  
+  `<INTERNAL_PORT>:<EXTERNAL_PORT>/TCP`
+  
+  For example:
+  
+  `8443:31594/TCP`
+  
+  The internal port is 8443 and the external port is 31594. When accessing the Alert UI the external port will be used in the URL.
+  Once the external port is identified the URL to access the UI will be in the following format:
+  
+  `https://my.kubernetes.host:<EXTERNAL_PORT>/alert`
```

Comment:
I believe this needs to be `https://<EXTERNAL_NODE_IP>:<EXTERNAL_PORT>/alert`. Users can get that information by doing a `kubectl get nodes -o wide` or by setting up an ingress.

Reply:
I will update this.